webrtc.html

```diff
@@ -4946,6 +4947,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn><code>upscaleAllowed</code></dfn> of type
+            <span class="idlMemberType"><a>boolean</a></span>,
+            defaulting to <code>false</code></dt>
+           <dd>
+             <p>When, and only when, this value is set true the browser MAY
+            upscale video temporally or spatially, or upscale the audio sampling
+            rate, or upscale the number of bits in the audio sample that this
+            <code><a>RtpSender</a></code> sends over RTP.
```

Comment:
Definitely included audio same rate upscale because we had a bunch of bugs with FF doing that breaking our ultrasonic audio pairing stuff we do.  But I don't think we talked about one control or more. What are you thinking with are use cases for separate policy on this?